docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,532 @@
+    --conf spark.pyspark.driver.python=/path/to/python \
+    --conf spark.pyspark.python=environment/bin/python \
+    train.py --cluster_mode bigdl-submit --remote_dir hdfs://url:port/path/to/remote/data
+```
+In the `bigdl-submit` script:
+* `--master`: the spark master, set it to yarn;
+* `--deploy-mode`: set it to client when running programs on yarn-client mode;
+* `--executor-memory`: set the memory for each executor;
+* `--driver-memory`: set the memory for the driver node;
+* `--executor-cores`: set the cores number for each executor;
+* `--num_executors`: set the number of executors;
+* `--py-files`: upload extra Python dependency files to YARN;
+* `--archives`: upload the Conda archive to YARN;
+* `--conf spark.pyspark.driver.python`: set the activate Python location on __Client Node__ as driver's Python environment;
+* `--conf spark.pyspark.python`: set the Python location in Conda archive as executors' Python environment;
+
+__Note:__
+* `--cluster_mode`: set the cluster_mode in `init_orca_context`.
+* `--remote_dir`: directory on a distributed storage for the dataset (see __Section 3__) and saving the model.
+* Please refer to __Section 4__ for the description of extra Python dependencies.
+* For CDH Users, please speficy `ARROW_LIBHDFS_DIR` (see __Section 2.3__) as a Spark conf in the script.
+    ```bash
+    bigdl-submit \
+    ...
+    --conf spark.executorEnv.ARROW_LIBHDFS_DIR=/opt/cloudera/parcels/CDH-${CHD_VERSION}/lib64/ \
+    train.py --cluster_mode bigdl-submit --remote_dir hdfs://url:port/path/to/remote/data
+    ```
+
+### 5.2.2 Yarn Cluster
+Submit and run the program on `yarn-cluster` mode following `bigdl-submit` script below: 
+```bash
+bigdl-submit \
+    --master yarn \
+    --deploy-mode cluster \
+    --executor-memory 10g \
+    --driver-memory 10g \
+    --executor-cores 8 \
+    --num-executors 2 \
+    --py-files model.py \
+    --archives /path/to/environment.tar.gz#environment \
+    --conf spark.yarn.appMasterEnv.PYSPARK_PYTHON=environment/bin/python \
+    --conf spark.executorEnv.PYSPARK_PYTHON=environment/bin/python \
+    train.py --cluster_mode bigdl-submit --remote_dir hdfs://url:port/path/to/remote/data
+```
+In the `bigdl-submit` script:
+* `--master`: the spark master, set it to `yarn`;
+* `--deploy-mode`: set it to `cluster` when running programs on yarn-cluster mode;
+* `--executor-memory`: set the memory for each executor;
+* `--driver-memory`: set the memory for the driver node;
+* `--executor-cores`: set the cores number for each executor;
+* `--num_executors`: set the number of executors;
+* `--py-files`: upload extra Python dependency files to YARN;
+* `--archives`: upload the Conda archive to YARN;
+* `--conf spark.yarn.appMasterEnv.PYSPARK_PYTHON`: set the Python location in Conda archive as Python environment of Application Master process;
+* `--conf spark.executorEnv.PYSPARK_PYTHON`: set the Python location in Conda archive as Python environment of executors.
```

Comment:
Add some description to say that the driver and executor will all use the archive for python environment